src/algebra/order/complete_field.lean

```diff
@@ -312,3 +312,26 @@ instance : unique (β ≃+*o γ) := unique_of_subsingleton $ induced_order_ring_
 
 end induced_map
 end linear_ordered_field
+
+section real
+
+/-- Any ring homomorphism `ℝ → ℝ` is an ordered ring isomorphism, and thus there are no
+nontrivial ring homorphism `ℝ → ℝ`. -/
+lemma real.ring_hom_eq_id (f : ℝ →+* ℝ) : f = ring_hom.id ℝ :=
```

Comment:
It w

Reply:
Done. Although I am not sure about the name...